test/index.test.js

```diff
@@ -2118,6 +2119,59 @@ describe('release-drafter', () => {
     })
   })
 
+  describe('with include-paths config', () => {
+    it('FIXME: returns the modified paths', async () => {
+      // FIXME: No idea how to actually write this test.
+      getConfigMock('config-with-include-paths.yml')
+
+      nock('https://api.github.com')
+        .post('/graphql', (body) =>
+          body.query.includes('query findCommitsWithAssociatedPullRequests')
+        )
+        .reply(200, graphqlCommitsMergeCommit)
+
+      nock('https://api.github.com')
+        .post('/graphql', (body) =>
+          body.query.includes('query findCommitsWithPathChangesQuery')
+        )
+        .reply(200, graphqlPathsMergeCommit)
+
+      nock('https://api.github.com')
+        .get(
+          '/repos/toolmantim/release-drafter-test-project/releases?per_page=100'
+        )
+        .reply(200, [])
+
+      //      nock('https://api.github.com')
```

Comment:
uncomment and use `yarn jest --updateSnapshot`

The test would write itself and data would be taken from the two query replies.